FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -45,6 +45,8 @@
 /// Notification when config is successfully activated
 const NSNotificationName FIRRemoteConfigActivateNotification =
     @"FIRRemoteConfigActivateNotification";
+static NSNotificationName RolloutsStateDidChangeNotificationName =
+    @"RolloutsStateDidChangeNotification";
```

Comment:
curious - should this be the same format as the other notification name? (prefixed with `FIR`)

Reply:
Ah yeah, since it's global let's perfix with `FIR`. Also this is in objc (objc does not have namespace so the best practise is uppercase prefixes for every object).